compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -97,7 +97,10 @@ void SchedulingOptions::bindOptions(OptionsBinder &binder) {
                      "internally but exporting functions as if synchronous."),
           clEnumValN(ExecutionModel::AsyncExternal, "async-external",
                      "Full HAL using asynchronous host/device execution both "
-                     "internally and externally.")),
+                     "internally and externally."),
+          clEnumValN(ExecutionModel::InlineStatic, "inline-static",
+                     "Inline host-local in-process execution with executable "
+                     "code statically linked into the host program.")),
```

Comment:
At some point we'll want to refresh https://iree-org.github.io/iree/deployment-configurations/bare-metal/ (or other pages) with information about the various execution models (async, inline) and output formats (VM / C)

Reply:
yep! want to give some time to trialing this first